sw/device/rom_exts/rom_ext_manifest.S

```diff
@@ -0,0 +1,85 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * ROM_EXT Manifest Format
+ */
+  .section .rom_ext_manifest, "a"
+
+  .globl rom_ext_image_start
+rom_ext_image_start:
+
+// Constant
+rom_ext_identifier:
+  .word 0x4552544F
+
+// Reserved
+  .word 0x0
+
+// Will be overidden by signing script
+image_signature:
+  .rept 96
+  .word 0x0
+  .endr
+
```

Comment:
Isn't there supposed to be 32bit break?

Reply:
No, a line that says `~~~  break  ~~~` just means that we're not showing the full length of that field to scale, because it's (comparatively) huge.